src/hooks/useThrottle.ts

```diff
@@ -0,0 +1,15 @@
+import { useCallback } from 'react';
+
+export const useThrottle = () => {
+  const throttle = useCallback(() => {
+    let timer: NodeJS.Timeout | null;
+    return (callback: () => void, delaySecond = 1) => {
+      if (timer) return;
```

Comment:
if문 중괄호 생략 가능하더라도 써주시면 좋을 것 같습니다 🤗
```suggestion
      if (timer) {
      return;
      }
```